src/_pytest/terminal.py

```diff
@@ -166,10 +166,13 @@ def getreportopt(config):
         reportchars = reportchars.replace("w", "")
     if reportchars:
         for char in reportchars:
-            if char not in reportopts and char != "a":
-                reportopts += char
-            elif char == "a":
+            if char == "a":
                 reportopts = "sxXwEf"
+            elif char == "A":
+                reportopts = "sxXwEfpP"
+                break
```

Comment:
Should we `break` on `if char == "a":` above?

Reply:
That would not work for `aA` then.. ;)

Reply:
Ahh right, thanks! 👍